cmd/common.go

```diff
@@ -256,36 +389,31 @@ func verifyBin(binPath []string) []string {
 	return w
 }
 
-func verifyKubeVersion(binPath []string) []string {
+func verifyKubeVersion(b string) []string {
 	// These executables might not be on the user's path.
 	// TODO! Check the version number using kubectl, which is more likely to be on the path.
 	var w []string
 
-	for _, b := range binPath {
-		_, err := exec.LookPath(b)
-		if err != nil {
-			w = append(w, fmt.Sprintf("%s: command not found on path - version check skipped\n", b))
```

Comment:
Let's keep this warning as part of the output if the command isn't found on the path - this is useful information to have even if --verbose is not on.